commands/core/docs.drush.inc

```diff
@@ -1,310 +0,0 @@
-<?php
-
-/**
- * @file
- *   Documentation commands providing various topics.
- */
-
-/**
- * Implementation of hook_drush_help().
- */
-function docs_drush_help($section) {
-  switch ($section) {
-    case 'meta:docs:title':
-      return dt('Documentation commands');
-    case 'meta:docs:summary':
-      return dt('Show information on various drush topics.');
-  }
-}
-
-/**
- * Implementation of hook_drush_command().
- *
- * @return
- *   An associative array describing your command(s).
- */
-function docs_drush_command() {
-  $docs_dir = drush_get_context('DOC_PREFIX', DRUSH_BASE_PATH);
-
-  //
-  // Topic commands.
-  // Any commandfile may add topics.
-  // Set 'topic' => TRUE to indicate the command is a topic (REQUIRED)
-  // Begin the topic name with the name of the commandfile (just like
-  // any other command).
-  //
-  $items['docs-readme'] = array(
-    'description' => 'README.md',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/README.md'),
-  );
-  $items['docs-bisect'] = array(
-    'description' => 'git bisect and Drush may be used together to find the commit an error was introduced in.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/examples/git-bisect.example.sh'),
-  );
-  $items['docs-bashrc'] = array(
-    'description' => 'Bashrc customization examples for Drush.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/examples/example.bashrc'),
-  );
-  $items['docs-configuration'] = array(
-    'description' => 'Configuration overview with examples from example.drushrc.php.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/examples/example.drushrc.php'),
-  );
-  $items['docs-config-exporting'] = array(
-    'description' => 'Drupal configuration export instructions, including customizing configuration by environment.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/docs/config-exporting.md'),
-  );
-  $items['docs-aliases'] = array(
-    'description' => 'Site aliases overview on creating your own aliases for commonly used Drupal sites with examples from example.aliases.drushrc.php.',
-    'hidden' => TRUE,
-    'topic' => TRUE,
-    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
-    'callback' => 'drush_print_file',
-    'callback arguments' => array($docs_dir . '/examples/example.aliases.drushrc.php'),
-  );
-  $items['docs-ini-files'] = array(
```

Comment:
I saw that we removed the example file but `\Drush\Commands\core\EditCommands::phpIniFiles` still supports `DRUSH_INI`, `/etc/drush.ini` and `~/.drush/drush.ini`.

Should we add `example.drush.ini` back?

Reply:
no, those files are no longer used. we need to fix EditCommands.

Reply:
See 2nd tip at http://docs.drush.org/en/master/install/

Reply:
So how are we going to update drush specific php config? Can we add php ini override in drush.yml?

Reply:
What is your use-case for drush-specific php config?

Reply:
To setup the memory limit `memory_limit = 256M`